.Gruntfile.babel.js

```diff
@@ -318,3 +325,38 @@ function script2ify (file) {
     cb(null, buf.toString('utf8').replace(regex, replacement))
   })
 }
+//
+var localeObject = {}
+try {
+  localeObject = JSON.parse(fs.readFileSync('./frontend/simple/locales/en/translation.json'))
+} catch (ex) {
+}
+function localify (file) {
+  return !/\.(vue|html|ejs)$/.test(file)
+    ? through()
+    : through(function (buf, encoding, cb) {
+      let functionEx = new RegExp(/([.|.$])L(\s*)\((\s*)(['|"])(\S*)(['|"])(\s*)([,|)])((\s*)(['|"])(\S*)(['|"])([,|)]))*/mg)
```

Comment:
- Can't this be done with just `/`? I don't think `new RegExp` is needed.
- No need for the beginning `$` since the function should just be `L`, and similarly no need for the check for the period at the start since the function should just be `import`ed.
- This regex looks complicated and possibly incorrect. One thing I notice is some `(\s*)` that don't need to be there, and things like `['|"]` which can be simplified to just `['"]`. Also the comment doesn't seem to be optional, which it is. Also, excessive/unnecessary capture groups seem to be used. I think this can be simplified down to something like:

```js
/\bL\(\s*['"](.*?)['"]\s*(?:,\s*['"](.*?)['"]\s*)?\)/mg
```

I haven't tested that, so it might not be perfect, but the gist is there. Similar simplifications are probably possible for the other regexps.